pyvistaqt/dialog.py

```diff
@@ -27,9 +27,7 @@ class FileDialog(QFileDialog):
     the dialog was property closed.
     """
 
-    # pylint: disable=too-few-public-methods
```

Comment:
Restoring this should make the Github Actions CI finally happy :)

Reply:
I'm on it!